scripts/reinstall-rapiddisk-b.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# For Debian: v1.01
+# Warning: ECC Memory recommended.
+# Reinstall Part 2/3 for Rapiddisk 8.2.0 Linux.
+# Everytime there is a Kernet change, Rapiddisk needs to be reinstalled.
+# To execute: save and `chmod +x ./reinstall-rapiddisk-b.sh` then `./reinstall-rapiddisk-b.sh`
+
+make
+sudo make install
```

Comment:
This assumes that you are in the root of the project directory. There needs to be some way to inform the user that this script needs to be there or some either change needs to occur.